tests/ExchangeRateTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Stripe;
+
+class ExchangeRateTest extends TestCase
+{
+    public function testRetrieve()
+    {
+        $this->mockRequest(
+            'GET',
+            '/v1/exchange_rates/usd',
+            array(),
+            array(
+                'id' => 'usd',
+                'object' => 'exchange_rates',
```

Comment:
Maybe switch this over to `exchange_rate` (and there are three other instances below to replace in this file as well).